.github/workflows/release.yml

```diff
@@ -195,7 +195,7 @@ jobs:
       - name: Smoke test
         shell: bash
         env:
-          RPC_URL: ${{ secrets.CHEATNET_RPC_URL }}
+          RPC_URL: ${{ secrets.RPC_URL }}
```

Comment:
Don't `secrets.RPC_URL` and `secrets.NODE_URL` have the same value?

Reply:
Forgot about that, I should have replaced it to `NODE_URL`. Good catch, thanks!